resources/views/user/studentProfile.blade.php

```diff
@@ -0,0 +1,157 @@
+				<div class="header">Course successfully added</div>
+				<div class="content">
+					<i class="check huge green circle icon"></i>
+					<p class="newCourseInfo" id="courseAddedName">You have been added to</p>
+					<p class="newCourseInfo" id="courseAddedName">Advanced Databases</p>
+				</div>
+				<div class="actions">
+					<div class="ui ok primary button">Done</div>
+				</div>
+			</div>
+			<div id="courseNotFound" class="ui modal">
+				<div class="header">Course not found</div>
+				<div class="content">
+					<i class="times huge red circle icon"></i>
+					<p class="newCourseInfo" id="courseAddedName">The course key you entered was not found.</p>
+				</div>
+				<div class="actions">
+					<div class="ui ok primary button">Done</div>
+				</div>
+			</div>
+			<div id="courseDuplicated" class="ui modal">
+				<div class="header">Duplicated course</div>
+				<div class="content">
+					<i class="times huge red circle icon"></i>
+					<p class="newCourseInfo" id="courseAddedName">You are already associated with this course.</p>
+				</div>
+				<div class="actions">
+					<div class="ui ok primary button">Done</div>
+				</div>
+			</div>
+		</div>
+	</div>
+@section( "scripts" )
+<script>
+	$(document).ready(function() {
+		// TODO: ajax to load student's current courses
+		$( "#courseKeyInputContainer" ).removeClass( "error" );
+		$( "#courseKey" ).removeClass( "error" );
+
+		$( ".coupled.modal" ).modal({
+            allowMultiple: false
+        });
+
+        $( ".second.modal" ).modal( "attach events", ".first.modal .button" );
+
+
+		$( "#addCourse" ).click(function() {
+			$( "#courseKeyInputContainer" ).removeClass( "error" );
+			$( "#courseKey" ).removeClass( "error" );
+
+			let courseKey = $( "#courseKey" ).val();
+
+			if(courseKey == "" ) {
+				$( "#courseKeyInputContainer" ).addClass( "error" );
+				$( "#courseKey" ).addClass( "error" );
+			}
+			else {
+				courseKey = courseKey.toUpperCase();
+				// TODO: ajax to check if course key exists
+				if(courseKey == "HELLO" || courseKey == "DOUBLE") {
+					// TODO: ajax to check course is not duplicated
+					if(courseKey == "HELLO") {
+						$( ".first.modal" ).modal({
+							transition: "fade up"
+						}).modal( "show" );
+					}
+					if(courseKey == "DOUBLE") {
+						$( "#courseDuplicated" ).modal({
+							transition: "fade up"
+						}).modal( "show" );
+					}
+				}
+				else {
+					$( "#courseNotFound" ).modal({
+						transition: "fade up"
+					}).modal( "show" );
+				}
+			}
+		});
+
+		$( "#confirmAddCourse" ).click(function() {
+			let courseKey = $( "#courseKey" ).val();
+
+			// TODO: ajax to associate student with course
+
+			let courseName = "Advanced Databases";
+			let courseProfessor = "Doctor Octopus";
+			let courseSchedule = "MoFri 13:00, Spring 2019";
+
+			let rowToAdd = "<tr>" + "<td>" + courseName + "</td>" + "<td>" + courseProfessor + "</td>" + "<td>" + courseSchedule + "</td>" + "</tr>";
```

Comment:
Ya podemos usar [modern string interpolation](https://medium.com/predict/switching-to-es6-part-2-string-interpolation-and-template-literals-2f1b0ee56740)

![Marie kondo](https://media1.giphy.com/media/uj9jW4Ba0dicHZYC1X/giphy.gif?cid=3640f6095c78a4596f3044454de483c1)